scifight/migrations/0012_create_fake_tournament.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+from django.db  import migrations
+from scifight   import models
+
+
+def create_fake_tournament(apps, schema_editor):
+    if models.Team.objects.count() > 0 or \
+                    models.Fight.objects.count() > 0 or \
+                    models.Jury.objects.count() > 0 or \
+                    models.JuryPoints.objects.count() > 0 or \
+                    models.Leader.objects.count() > 0 or \
+                    models.LeaderToJury.objects.count() > 0 or \
+                    models.Participant.objects.count() > 0 or \
+                    models.Problem.objects.count() > 0 or \
+                    models.Refusal.objects.count() > 0 or \
+                    models.Room.objects.count() > 0:
+        fake_tournament = models.Tournament(name="--- FAKE MIGRATION TOURNAMENT, DELETE ASAP ---")
+        fake_tournament.save()
+        models.Team.objects.update(tournament=fake_tournament)
+        models.Fight.objects.update(tournament=fake_tournament)
+        models.Jury.objects.update(tournament=fake_tournament)
+        models.JuryPoints.objects.update(tournament=fake_tournament)
+        models.Leader.objects.update(tournament=fake_tournament)
+        models.Participant.objects.update(tournament=fake_tournament)
+        models.Problem.objects.update(tournament=fake_tournament)
+        models.Refusal.objects.update(tournament=fake_tournament)
+        models.Room.objects.update(tournament=fake_tournament)
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('scifight', '0011_add_model_tournament'),
+    ]
+
+    operations = [
+        migrations.RunPython(
+            create_fake_tournament,
+            reverse_code=migrations.RunPython.noop),
+    ]
```

Comment:
I think at this point we may skip writing save non-destructive migrations as we're going to throw them away at `1.0-alpha` milestone.